### ISSUE TITLE
Japanese charactes of examples doesn't works

### ISSUE THREAD

**Issue by olidroide** [jezeniel/smsutil#2]
Hi, i'm testing your lib and trying to test with your example but I receive these error:

```>>> utf16 = smsutil.encode('最高でした 🍔')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/smsutil/__init__.py", line 36, in encode
    return text.encode(encoding)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 0: ordinal not in range(128)
>>> sms = smsutil.split('[the quick brown fox]')
>>> len(sms.parts)
1
>>> sms = smsutil.split('最高でした 🍔')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python2.7/dist-packages/smsutil/__init__.py", line 131, in split
    return unicode_split(text)
  File "/usr/local/lib/python2.7/dist-packages/smsutil/__init__.py", line 100, in unicode_split
    message += char
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe6 in position 0: ordinal not in range(128)```

**Comment by jezeniel** [comment]
@olidroide i think this is an issue with python 2.7, will look into this and push a fix. The easiest work around is to do `sms = smsutil.split(u'最高でした 🍔')` with a `u` at the beginning.

**Comment by jezeniel** [comment]
Fixed on 48deae19a225e
